### DISCUSSION TITLE
Is it possible to add products?

### DISCUSSION THREAD

**Discussion by arkilis** [tinacms/tinacms#4533] [Experimental]
Such as having product list using stripe and selling online?

**Comment by guillermodlpa** [comment]
This can be done already on top of Tina. We've implemented this ourselves in different ways:

- Making a block called "Products", that has configurable products in it and displays them as a list.
- Making a collection called "products", with each product having page.

Then, each product has a field for the Stripe product ID, and using the Stripe JS library, we create a checkout page and redirect to it when the user clicks the purchase button.
